hello.yml

```diff
@@ -9,4 +9,4 @@ config:
 scenarios:
   - flow:
     - get:
-        url: "/recipes"
\ No newline at end of file
+        url: "/recipes/id/5beca9968d94d447f4de2cb1"
```

Comment:
this doesn't look like a valid recipe id, it should be a 5 or 6 digit numeric id